fabrics.c

```diff
@@ -744,6 +740,9 @@ int nvmf_discover(const char *desc, int argc, char **argv, bool connect)
 		OPT_INCR("verbose",      'v', &verbose,       "Increase logging verbosity"),
 		OPT_FLAG("dump-config",  'O', &dump_config,   "Dump configuration file to stdout"),
 		OPT_FLAG("force",          0, &force,         "Force persistent discovery controller creation"),
+		OPT_FLAG("nbft",           0, &nbft,          "Only look at NBFT tables"),
+		OPT_FLAG("no-nbft",        0, &nonbft,        "Do not look at NBFT tables"),
+		OPT_STRING("nbft-path",  0, "STR", &nbft_path, "user-defined path for NBFT tables"),
```

Comment:
Looks about right. Personally I would have gone for `nbft-only`, `nbft-ignore` and `nbft-path`. But I can live with this. Anyway, it would be cool if you could also update the documentation (just the `*.txt` file, the rest will be automatically regenerated on release anyway)

@hreinecke are you also fine with this?

Reply:
Resolving this thread.